### ISSUE TITLE
accessibility issue 20 (low) - make facets into buttons using role attribute

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#254]
(1)	The “links” in the search filter should really be buttons (Collection, Creator, Date, On-site Access, Rights, Image Available).
(2)	These buttons should have an aria-expanded attribute of true or false depending on the state of the button.  This attribute is appropriate for button elements.
ARIA Authoring Practices for Disclosure buttons: 
https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/3660011dd7c0a32d42c8a1c4a7ef56e609e39695
